web/src/pages/index.md

```diff
@@ -7,35 +7,55 @@ title: ECE 477 Team 5 Homepage
+of hall effect sensors, which pinpoint the puck's every move. Additionally,
+photoresistors are integrated for impeccable goal detection, ensuring each score
+is accurately registered.
+
+The hall effect sensors are strategically embedded beneath the translucent
+playing field in a grid pattern. These sensors detect changes in magnetic
+fields. As the puck, equipped with a small magnet, glides across the surface,
+the sensors underneath identify its precise location by sensing the magnetic
+field changes caused by the puck's movement. The density and layout of this grid
+ensure the puck's position is determined with high accuracy, allowing the system
+to provide real-time feedback. For instance, as the puck moves, the individually
+controlled LEDs beneath the surface illuminate its path, creating a mesmerizing
+trail effect that enhances the overall gaming experience.
+
+This grid-based hall effect sensor system not only offers unparalleled precision
+in puck tracking but also enables the microcontroller to interpret rapid changes
+in direction and speed. Such granular detail ensures that dynamic visuals
+generated by the LEDs are always synchronized with the puck's position and
+movement. Players, therefore, experience a highly responsive and visually rich
+gameplay, where the table reacts instantly to every twist and turn of the puck.
+
+Combined with other features like the OLED score display and photoresistor-based
+goal detection, the SAHT offers an intuitive yet futuristic air hockey
+experience. It seamlessly merges traditional gameplay with modern tech
+enhancements, ensuring that both newcomers and seasoned players are treated to
+an unmatched recreational encounter.
 
 ## Project Specific Design Requirements (PSDRs)
 
 The design and development of the smart air hockey table will adhere to the
 following focused design requirements to ensure a successful and captivating
 gameplay experience:
 
-1. PSDR #1 (Hardware):
-   An ability to convert analog hall effect sensor readings into digital signals with custom threshold values using comparators.
-2. PSDR #2 (Hardware):
-   An ability to compress digital readings from a grid of hall effect sensors to simplified row and column position data using logic gates.
-3. PSDR #3 (Hardware):
-   An ability to reliably detect goals scored by monitoring the passage of the puck through the goal using a photoresistor and LED pair.
-4. PSDR #4 (Hardware):
-   An ability to display game score on an OLED display using the SPI protocol.
-5. PSDR #5 (Software):
-   An ability to control a grid of individually-addressable RGB LEDs beneath the playing surface using a serialized PWM communication protocol.
+1. PSDR #1 (Hardware): An ability to efficiently and quickly compress the
+   signals from a large grid of hall effect switches under the playing surface
+   using digital gates to just row and column bit vectors. These are connected
+   to the microcontroller and used to determine the puck position.
```

Comment:
These seem way too detailed.  He said we can be a bit more vague and then clarify it in the project description

Reply:
I think being as simple as "An ability to determine the position of an air hockey puck using under-table hall effect sensors and digital logic gates." might work, as long as we describe how we do that in the description.

Reply:
I believe he said either or worked. I already stripped it down as much as it made sense s.t. someone can just look at it and know what we want exactly

Reply:
But being this detailed makes it equally difficult to know what we want.  Average user doesn't know how we use logic gates to get bit vectors, but they can understand getting "puck position"

Reply:
okay :(